spec/features/user_edits_profile_spec.rb

```diff
@@ -25,8 +25,7 @@
     fill_in "user[last_name]", with: "Abriel"
     fill_in "user[age]", with: "23"
     fill_in "user[website]", with: "http://www.abhnation.com/"
-    fill_in "user[current_password]", with: user.password
-    click_button "Update"
+    first('#account > div:nth-child(4) > div').click_button "Update"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.